doc/quick-start-guide.md

```diff
@@ -162,9 +161,6 @@ We should see some output similar to this:
     - docker
         - status: present
         - version info: Docker version 19.03.6, build 369ce74a3c
-    - docker-compose
```

Comment:
From the changes in `doctor` we're still presenting a `docker compose` entry (just removing the hyphen)

Reply:
Although if there's no `docker compose` it will check for  `docker-compose`. I guess just undoing this deletion is just fine (even keeping it is OK, we just want to send the literal report)